src/background/advisory/debricked.js

```diff
@@ -61,12 +61,16 @@ export default async ({ type, name }) => {
   const { metrics: packageMetrics } = await getOshData(dependencyId);
 
   let issues = 0;
+  let summaries = [];
```

Comment:
this can be a const it is not changing.
```suggestion
  const summaries = [];
```

Reply:
Oh, thanks for reminding me!
I noticed this in another file, and I said wait, I should not care about such things, it needs to be forced by ESLint.

Do you want to add it to the ESLint rules, or I will do it myself?

Reply:
#69